src/condor_includes/sock.h

```diff
@@ -233,6 +234,23 @@ class Sock : public Stream {
         // Encryption support below
         //------------------------------------------
         bool set_crypto_key(bool enable, KeyInfo * key, const char * keyId=0);
+
+        int ciphertext_size(int plaintext_size) {
+            dprintf(D_ALWAYS, "ZKM: SOCK:CIPHERSIZE(): crypto_ is %p.\n", crypto_);
+            if (!crypto_) return plaintext_size;
+            dprintf(D_ALWAYS, "ZKM: SOCK:CIPHERSIZE(): crypto_state_ is %p.\n", crypto_state_);
+            if (!crypto_state_) return plaintext_size;
+            dprintf(D_ALWAYS, "ZKM: SOCK:CIPHERSIZE(): crypto_state_->m_keyInfo is %p.\n", (void*)&crypto_state_->m_keyInfo);
+            dprintf(D_ALWAYS, "ZKM: SOCK:CIPHERSIZE(): crypto_state_->m_keyInfo.protocol_ %i.\n", crypto_state_->m_keyInfo.getProtocol());
+
+            // for all methods other than AESGCM just return plaintext
+            if (crypto_state_->m_keyInfo.getProtocol() != CONDOR_AESGCM) return plaintext_size;
```

Comment:
There's a virtual function (`ciphertest_size_with_cs`) that does this for us, no?  Why is the logic leaked to here?

Reply:
Agreed, I think I just need to update call sites to use the new function.